tests/controllers/UsersControllerTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace JoindinTest\Controller;
+
+
+class UsersControllerTest extends \PHPUnit_Framework_TestCase
+{
+
+    /**
+     * Ensures that if the deleteUser method is called and no user_id is set,
+     * an exception is thrown
+     *
+     * @return void
+     *
+     * @test
+     * @expectedException        \Exception
+     * @expectedExceptionMessage You must be logged in to delete data
+     */
+    public function testDeleteUserWithNoUserIdThrowsException()
+    {
+        $request = new \Request([], ['REQUEST_URI' => "http://api.dev.joind.in/v2.1/users/3", 'REQUEST_METHOD' => 'DELETE']);
+
+        $usersController = new \UsersController();
+        $db = $this->getMockBuilder('\JoindinTest\Inc\mockPDO')->getMock();
```

Comment:
You could also use ```$this->getMockBuilder('PDO')->disableOriginalConstructor()->getMock();``` to Mock the PDO-Instance. Pro *and* Con is you need to mock the called methods as well. :wink: